example/cumulus-tf/variables.tf

```diff
@@ -475,3 +475,42 @@ variable "dead_letter_recovery_memory" {
   default = 1024
   description = "The amount of memory in MB to reserve for the dead letter recovery Async Operation Fargate Task"
 }
+
+variable "workflow_configurations" {
+  description = <<EOF
```

Comment:
Nice docstring - we should go through and do this sort of thing for most of our configurations.